.github/workflows/scripts/before_install.sh

```diff
@@ -122,7 +122,7 @@ fi
 cd ..
 
 
-git clone --depth=1 https://github.com/pulp/pulp_ansible.git --branch 0.13
+git clone --depth=1 https://github.com/pulp/pulp_ansible.git --branch 0.13.0
```

Comment:
Should the CI tooling related changes go into a separate PR?

Reply:
moved to https://github.com/ansible/galaxy_ng/pull/1307